contracts/SimpleSwapFactory.sol

```diff
@@ -16,18 +17,23 @@ contract SimpleSwapFactory {
 
   /* address of the ERC20-token, to be used by the to-be-deployed chequebooks */
   address public ERC20Address;
+  /* address of the code contract from which all chequebooks are cloned */
+  address public master;
 
-  constructor(address _ERC20Address) public {
+  constructor(address _ERC20Address, address _master) public {
     ERC20Address = _ERC20Address;
+    master = _master;
   }
   /**
   @notice deployes a new SimpleSwap contract
```

Comment:
Typo here (deployes) and let's adjust the comment, specifying that we create a clone

Reply:
fixed